pkgs/applications/misc/streamdeck-ui/default.nix

```diff
@@ -11,13 +11,13 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "streamdeck-ui";
-  version = "2.0.4";
+  version = "2.0.6";
 
   src = fetchFromGitHub {
     repo = pname;
     owner = "timothycrosley";
     rev = "v${version}";
-    hash = "sha256-NV4BkHEgfxIOuLfmn0vcPNqivmHLD6v7jLdLZgnrb0Q=";
+    sha256 = "sha256-5dk+5oefg5R68kv038gsZ2p5ixmpj/vBLBp/V7Sdos8=";
```

Comment:
nit: I had thought `hash` was preferred when using this format? It looks like it doesn't matter now, because the build fetches the source correctly.

Reply:
I use lib.fakeShaXXX for getting the new hashes which wasn't working with hash why I changed it.